rpc/handler.go

```diff
@@ -507,7 +507,7 @@ func (h *handler) handleCallMsg(ctx *callProc, msg *jsonrpcMessage) *jsonrpcMess
 			if resp.Error.Data != nil {
 				logctx = append(logctx, "errdata", formatErrorData(resp.Error.Data))
 			}
-			h.log.Warn("Served "+msg.Method, logctx...)
+			h.log.Debug("Served "+msg.Method, logctx...)
```

Comment:
did we find why these logs were populated after the migration?
TBH converting warn to debug does not seem right to me

Reply:
but okay with this change, just need to keep the above point, about non-heimdall calls failing, into mind.